ddtrace/_trace/utils_botocore/span_pointers/telemetry.py

```diff
@@ -0,0 +1,26 @@
+from ddtrace.internal.telemetry import telemetry_writer
+
+
+def record_span_pointer_calculation(span_pointer_count: int) -> None:
+    telemetry_writer.add_count_metric(
+        namespace="tracer",
+        name="botocore_span_pointer_calculation",
```

Comment:
should this instead be a more generic `span_pointer_calculation` metric, with something like `library:botocore` ?

Reply:
i've moved the code to a more generic metric approach